lib/content-services/src/lib/dialogs/dialog/dialog.component.scss

```diff
@@ -28,191 +29,201 @@ $dialog-padding: 24px;
+        padding: 0;
+    }
 
-.adf-dialog-header {
-    position: relative;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding-bottom: $dialog-padding;
+    .adf-dialog-header {
+        position: relative;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding-bottom: $dialog-padding;
+
+        &.adf-centered-header {
+            align-items: flex-start;
+        }
 
-    &.adf-centered-header {
-        align-items: flex-start;
-    }
+        .adf-dialog-title {
+            font-size: large;
+            font-weight: 200;
+            margin: 0;
+        }
 
-    .adf-dialog-title {
-        font-size: large;
-        font-weight: 200;
-        margin: 0;
+        .adf-dialog-header-icon {
+            width: 48px;
+            height: 48px;
+            font-size: 48px;
+            color: $theme-blue-button-color;
+        }
     }
 
-    .adf-dialog-header-icon {
-        width: 48px;
-        height: 48px;
-        font-size: 48px;
-        color: $theme-blue-button-color;
+    .adf-description {
+        padding-top: $dialog-padding;
     }
-}
 
-.adf-description {
-    padding-top: $dialog-padding;
-}
+    .adf-dialog-content {
+        margin: $dialog-padding 0;
+        overflow: auto;
+        flex: 1;
+        max-height: none;
+        line-height: 20px;
+    }
 
-.adf-dialog-content {
-    margin: $dialog-padding 0;
-    overflow: auto;
-    flex: 1;
-    max-height: none;
-    line-height: 20px;
-}
+    .adf-dialog-header,
+    .adf-dialog-actions {
+        &::after {
+            content: ' ';
+            position: absolute;
+            left: -$dialog-padding;
+            display: block;
+            height: 0;
+            border-bottom: 1px solid #2121211f;
+            width: calc(100% + $dialog-padding * 2);
+        }
+    }
 
-.adf-dialog-header,
-.adf-dialog-actions {
-    &::after {
-        content: ' ';
-        position: absolute;
-        left: -$dialog-padding;
-        display: block;
-        height: 0;
-        border-bottom: 1px solid #2121211f;
-        width: calc(100% + $dialog-padding * 2);
+    .adf-dialog-actions::after {
+        top: 0;
     }
-}
 
-.adf-dialog-actions::after {
-    top: 0;
-}
+    .adf-dialog-header::after {
+        bottom: 0;
+    }
 
-.adf-dialog-header::after {
-    bottom: 0;
-}
+    .adf-dialog-actions {
+        position: relative;
+        display: flex;
+        justify-content: space-between;
+        padding-top: $dialog-padding;
+        text-transform: capitalize;
+        min-height: auto;
 
-.adf-dialog-actions {
-    position: relative;
-    display: flex;
-    justify-content: space-between;
-    padding-top: $dialog-padding;
-    text-transform: capitalize;
-    min-height: auto;
+        // TODO: Update after migration to Angular 15
+        /* stylelint-disable-next-line selector-class-pattern */
+        .mat-button.mat-button-base + .mat-button-base {
```

Comment:
I remember we were trying to reduce/remove the usages of angular material classes in our code base. Is the TODO comment for that only? Just confirming

Reply:
We'll try to remove it, but we might still need it because in the design we need to have a different margin (from `8px` to `16px`) between the buttons and a different `border-radius` (from `4px` to `8px`) of the container.